# disables some css which sb-admin-2 and sb-admin-2.css and execution of npm run minimize:
# details are included in: https://github.com/BlackrockDigital/startbootstrap/issues/434

# The frontend build here includes a dependency that doesn't exist anymore on the internet, 
# Thankfully it's just some js and css, this step disables that part. We should check to see if this is safe to remove after the next release, 
# the project will have to change the library here significantly.

diff --git a/src/server/pom.xml b/src/server/pom.xml
index 2962017e..2f0e80eb 100644
--- a/src/server/pom.xml
+++ b/src/server/pom.xml
@@ -605,21 +605,6 @@
                             <goal>exec</goal>
                         </goals>
                     </execution>
-                    <execution>
-                        <id>exec-npm-run</id>
-                        <phase>generate-sources</phase>
-                        <configuration>
-                            <executable>npm</executable>
-                            <workingDirectory>../ui</workingDirectory>
-                            <arguments>
-                                <argument>run</argument>
-                                <argument>minimize</argument>
-                            </arguments>
-                        </configuration>
-                        <goals>
-                            <goal>exec</goal>
-                        </goals>
-                    </execution>
                 </executions>
             </plugin>
             <plugin>
diff --git a/src/ui/bower.json b/src/ui/bower.json
index eef3fea8..57d576a5 100644
--- a/src/ui/bower.json
+++ b/src/ui/bower.json
@@ -20,7 +20,6 @@
   ],
   "devDependencies": {
     "jquery": "~2.1.3",
-    "startbootstrap-sb-admin-2": "https://github.com/BlackrockDigital/startbootstrap-sb-admin-2.git#v3.3.7+1",
     "rxjs": "~2.4.6",
     "moment": "~2.9.0",
     "datatables.net": "^1.10.16",
diff --git a/src/ui/package.json b/src/ui/package.json
index 0be5cd98..b681ae3c 100644
--- a/src/ui/package.json
+++ b/src/ui/package.json
@@ -27,7 +27,7 @@
     "jsx-loader": "^0.13.0",
     "lodash": "^4.17.0",
     "moment": "^2.22.2",
-    "node-sass": "^4.9.0",
+    "node-sass": "^8.0.0",
     "react": "^16.13.0",
     "react-bootstrap": "^0.33.0",
     "react-dom": "^16.12.0",
diff --git a/src/ui/webpack.config.js b/src/ui/webpack.config.js
index 7d5438c2..c68df5eb 100644
--- a/src/ui/webpack.config.js
+++ b/src/ui/webpack.config.js
@@ -19,7 +19,6 @@ var webpack = require('webpack');
 var HtmlWebpackPlugin = require('html-webpack-plugin');
 
 var _commonDeps = [
-  "sb-admin-2.css",
   "bootstrap.css",
   "theme.css",
   "font-awesome.css",
@@ -29,7 +28,6 @@ var _commonDeps = [
   "react",
   "bootstrap",
   "metisMenu",
-  "sb-admin-2",
   "rxjs",
   "bootstrap-multiselect.css",
   "datatables.net-bs.css",
@@ -86,10 +84,8 @@ module.exports = {
       "jquery": "jquery/dist/jquery",
       "bootstrap.css": "bootstrap.css",
       "theme.css": "bootstrap-theme.css",
-      "sb-admin-2.css": "startbootstrap-sb-admin-2/dist/css/sb-admin-2.css",
       "font-awesome.css": "font-awesome/css/font-awesome.min.css",
       "metisMenu.css": "metisMenu/dist/metisMenu.min.css",
-      "sb-admin-2": "startbootstrap-sb-admin-2/dist/js/sb-admin-2.js",
       "rxjs": 'rxjs/dist/rx.all',
       "moment": 'moment/moment',
       "datatables.net": "datatables.net/js/jquery.dataTables.js",